src/hooks/usePhaseData.ts

```diff
@@ -121,34 +119,34 @@ export const usePhaseData = (phaseEnum?: PhaseEnum | null): GetPhaseData => {
 
   // activityTypes
   // {INITIATION_SPECIAL_GIFT_APPEAL: {
-  //   name: 'Special Gift Appeal',
+  //   translatedShortName: 'Special Gift Appeal',
+  //   translatedFullName: 'Initiation - Special Gift Appeal',
   //   phase: 'Initiation',
   //   phaseId: 'INITIATION',
   //   subject: 'Special Gift Appeal',
-  //   title: 'Initiation - Special Gift Appeal',
   // },
   // FOLLOW_UP_TEXT_MESSAGE: {
-  //   name: 'Text Message',
+  //   translatedShortName: 'Text Message',
+  //   translatedFullName: 'Follow Up - Text Message',
   //   phase: 'Follow-Up',
   //   phaseId: 'FOLLOW_UP',
   //   subject: 'Text Message To Follow Up',
-  //   title: 'Follow Up - Text Message',
   // },}
 
-  const activityTypes: Map<ActivityTypeEnum, ActivityData> = useMemo(() => {
-    const activitiesMap = new Map();
+  const activityTypes = useMemo(() => {
+    const activitiesMap = new Map<ActivityTypeEnum, ActivityData>();
 
     constants?.phases?.forEach((phase) => {
       phase?.tasks?.forEach((activityType) => {
         const activity = constants?.activities?.find(
           (activity) => activity.id === activityType,
         );
         activitiesMap.set(activityType, {
-          name: getLocalizedTaskType(t, activityType),
+          translatedFullName: activity?.value || '',
+          translatedShortName: activity?.action || '',
```

Comment:
Sorry, I'm going to go back on this. I just noticed that everywhere that uses `translatedFullName` has to do `activitiesMap.get()?.translatedFullName` and since `get()` can return undefined, it needs a default value anyway. So it doesn't actually help to add the default value here. So sorry for the back and forth.

```suggestion
          translatedFullName: activity?.value,
          translatedShortName: activity?.action,
```